### ISSUE TITLE
Missing instructions on components that need to be moved to new compute unit

### ISSUE THREAD

**Issue by mhauke** [NetAppDocs/storagegrid-appliances#20]
### Page URL

https://docs.netapp.com/us-en/storagegrid-appliances/sg6100/replacing-sg6100-cn-controller.html

### Page title

Replace SG6100-CN controller

### Summary

On this page it simply states take out the old compute node and put the new one into the rack. I believe there should be mentioned that some components do not come with the compute node chassis FRU and need to be moved from the old compute node to the new one, like NICs, PSUs, & SSDs. 

### Public issues must not contain sensitive information

- [X] This issue contains no sensitive information.

**Comment by netapp-pcarriga** [comment]
Hi @mhauke, thanks for this suggestion. I'll investigate and update you here.

**Comment by mmears** [comment]
Agreed, this is a miss that needs to be rectified.  The chassis replacement instructions for SGF6112 refer to moving the FRU parts from the old chassis to the replacement.  The instructions for SG6100-CN do not, but should.  The two systems have very similar FRU part lists since they are based on a common design center (SG6100-CN has one more NIC, and fewer SSDs).
